tutorial/mfxx49820_bayesian_opt.yaml

```diff
@@ -0,0 +1,98 @@
+setup:
+  queue: 'anaq'
+  root_dir: '/cds/data/drpsrcf/mfx/mfxx49820/scratch/btx/'
+  exp: 'mfxx49820'
+  run: 15
+  det_type: 'epix10k2M'
+
+fetch_mask:
+  dataset: '/data/data'
+
+fetch_geom:
+
+build_mask:
+  thresholds: -10 5000
+  n_images: 20
+  n_edge: 1
+  combine: True
+
+run_analysis:
+  max_events: -1
+
+opt_geom:
+  n_iterations: 5
+  n_peaks: 3
+  threshold: 1000000
+  center: 823 833
+
+find_peaks:
+  tag: ''
+  psana_mask: False
+  min_peaks: 10
+  max_peaks: 2048
+  npix_min: 2
+  npix_max: 30
+  amax_thr: 40.
+  atot_thr: 180.
+  son_min: 10.0
+  peak_rank: 3
+  r0: 3.0
+  dr: 2.0
+  nsigm: 10.0
+
+index:
+  time: '3:00:00'
+  ncores: 64
+  tag: 'sample2'
+  int_radius: '3,4,5'
+  methods: 'mosflm'
+  tolerance: '5,5,5,1.5'
+  no_revalidate: True
+  multi: True
+  profile: True
+  cell: '/cds/data/drpsrcf/mfx/mfxx49820/scratch/btx/cell/sample2.cell'
+
+stream_analysis:
+  tag: 'sample2'
+
+determine_cell:
+  tag: 'sample2'
+
+merge:
+  tag: 'sample2'
+  symmetry: '4/mmm_uac'
+  iterations: 1
+  model: 'unity'
+  foms: 'CCstar Rsplit'
+  nshells: 10
+  highres: 1.6
+
+bayesian_optimization:
```

Comment:
Elsewhere (also commented) this has been named differently (`bayesian_opt`) - I think either is fine, maybe with a preference for the full spelling, but we should just make sure things are consistent.

Reply:
The name "bayesian_optimization" in the .yaml refers to the "bayesian_optimization" task in tasks.py. This should be correct.